docs/source/couchdb_tutorial.rst

```diff
@@ -930,27 +911,26 @@ line curl utility to create and update indexes.
 .. note:: The Fauxton interface is a web UI for the creation, update, and
```

Comment:
I know this was here before, but do we really need to have explicit doc on how to use CouchDB tools?

Reply:
From past experience, Fabric users are not usually experienced database admins and may need a push in the right direction. See the best practices for queries and indexes section above, which was put in based on user feedback. It also makes the tutorial a bit more fun. You can go to your browser and see your indexes. The note is far down enough that it does not clutter the tutorial.

Reply:
ok